proposals/0000-template.md

```diff
@@ -83,6 +83,37 @@ learnability of the language for novice users. Define and list any remaining
 drawbacks that cannot be resolved.
 
 
+## Backward Compatibility
+
+Will your proposed change cause any existing programs to change behaviour or
+stop working? Assess the expected impact on existing code on the following scale:
+
+0. No breakage
+1. Breakage only in extremely rare cases (e.g. for specifically-constructed
+   examples, but probably no Hackage packages)
+2. Breakage in rare cases (e.g. a few Hackage packages may break, but probably
+   no Stackage packages)
```

Comment:
It's not clear to me what the difference between Hackage and Stackage is.  Spell it out!

Reply:
I adjusted this a little to hopefully be clearer. I'm not sure it makes sense to give a full account of the relationship between Hackage and Stackage.